### DISCUSSION TITLE
why does not support stream_proxy like Apisix?

### DISCUSSION THREAD

**Discussion by JamesBonddu** [Kong/kong#13927] [Help]
why does not support stream_proxy like Apisix.

https://github.com/apache/apisix/blob/a0b1a4554dc9ce23e877cc1fc58507c795203878/docs/en/latest/stream-proxy.md

**Comment by xianghai2** [comment]
Kong do support tcp/tls proxy, right ? https://docs.konghq.com/gateway/latest/how-kong-works/routing-traffic/#proxy-tcptls-traffic
Check if this could meet your needs.

Thanks.
